src/dictionaries/sv.json

```
@@ -179,5 +180,62 @@
   },
   "about": {
     "title": "Om"
+  },
+  "settings": {
+    "common": {
+      "controlPanel": "Kontrollpanel",
+      "priority": "Sorteringsprioritet",
+      "name": "Namn",
+      "nameEn": "Namn (Engelska)",
+      "nameSv": "Namn (Svenska)",
+      "type": "Typ",
+      "actions": "Åtgärder",
+      "category": "Kategori"
+    },
+    "general": {
+      "name": "Allmänt",
+      "user": "Användare",
+      "theme": "Tema",
+      "loggedIn": "Inloggad som",
+      "notLoggedIn": "Inte inloggad",
+      "themeResolved": "Aktuellt tema löst till",
+      "systemTheme": "Temapreferens löst till",
```

```suggestion
      "themeResolved": "Aktuellt tema",
      "systemTheme": "Temapreferens",
```

Is this better? Don't really like "löst till".